src/duckling/project/asset.service.ts

```diff
@@ -113,7 +113,16 @@ export class AssetService {
     }
     
     private _getFilePath(assetToLoad : LoadingAsset) : string {
```

Comment:
Is this an optional arg? If so can we type it that way

Reply:
I don't think so, every caller of _getFilePath uses it